index.html

```diff
@@ -363,36 +363,83 @@ <h2>Security Considerations</h2>
-    of all input data. It is essential that any JSON inputs that do not conform to the JSON-text syntax defined in [[rfc7159]]
-    be rejected in their entirety by JSON parsers. Failure to reject invalid input could compromise 
-    the security and integrity of Verifiable Credentials.
-  </p>
+  <section>
+    <h2>General Considerations</h2>
+    <p>
+      When implementing VC-JWTs, it is essential to address all security issues relevant 
+      to broad cryptographic applications. This especially includes protecting the user's
+      asymmetric private and symmetric secret keys, as well as employing countermeasures 
+      against various attacks. Failure to adequately address these issues could compromise 
+      the security and integrity of Verifiable Credentials, potentially leading to unauthorized 
+      access, modification, or disclosure of sensitive information. 
+    </p>
+    <p>
+      Implementers are advised to follow best practices and established cryptographic standards to ensure 
+      the secure handling of keys and other sensitive data. 
+      Additionally, conduct regular security assessments and audits to identify and 
+      address any vulnerabilities or threats.
+    </p>
+    <p>
+      Follow all security considerations outlined in [[rfc7515]] and [[rfc7519]].
+    </p>
+    <p>
+      When utilizing JSON-LD, take special care around remote retrieval of contexts and follow the
+      additional security considerations noted in [[json-ld11]].
+    </p>
+    <p>
+      As noted in [[rfc7515]] when utilizing JSON [[rfc7159]], strict validation is a security
+      requirement. If malformed JSON is received, it may be impossible to reliably interpret the 
+      producer's intent, potentially leading to ambiguous or exploitable situations.
+      To prevent these risks, it is essential to use a JSON parser that strictly validates the syntax 
+      of all input data. It is essential that any JSON inputs that do not conform to the JSON-text syntax defined in [[rfc7159]]
+      be rejected in their entirety by JSON parsers. Failure to reject invalid input could compromise 
+      the security and integrity of Verifiable Credentials.
+    </p>
+  </section>
+  <section>
+    <h2>Context Integrity</h2>
+    <p>
+      In some cases is is desireable to know that the version of the
+      context(s) utilized in the verifiable credential is the same as
+      used by both the issuer and verifier.  
+    </p>
+    <p>
+      To validate that a context included in a Verifiable Credential is
+      the same at verification time as at issuing time an implementer
+      MAY include a property named <code>contextIntegrity</code> that
+      stores an array of objects that describe additional integrity
+      metadata about the context. Each object in the
+      <code>contextIntegrity</code> array MUST contain the following:
+      the URL to the context named <code>context</code>, a
```

Comment:
Currently, there might be two elements within `contextIntegrity` with the same `context` value. Should there be guidance that only one may be present?